postgres-appliance/Dockerfile.build

```diff
@@ -328,7 +334,13 @@ RUN export DEBIAN_FRONTEND=noninteractive \
         && if [ -d /usr/share/postgresql/9.5/extension ]; then \
             cd /usr/share/postgresql/9.5/extension \
             && for f in *.sql *.control; do \
-                for v in 9.3 9.4 9.6 10; do \
+                  if [ -L /usr/share/postgresql/9.5/extension/$f ]; then \
+                     continue; \
+                  fi; \
+                  for v in $PGOLDVERSIONS; do \
+                      if [ $v == 9.5 ]; then \
```

Comment:
Let's combine it with the next if block:
`if [ $v != 9.5 ] && [ -f /usr/share/postgresql/$v/extension/$f ] \`